src/parser/Parser.ts

```diff
@@ -1114,6 +1123,36 @@ export class Parser {
             return new Stmt.Return(tokens, toReturn);
         }
 
+        /**
+         * Parses a `goto` statement
```

Comment:
Looks like a bit of a copy-pasta error in the docs for `labelStatement` 😃

Reply:
Fixed!

Reply:
Good catch btw.